lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -228,6 +235,59 @@ public function execute($params = null)
         return true;
     }
 
+    /**
+     * Split $this->_bindedValues into those values that need to be sent using mysqli::send_long_data()
+     * and those that can be bound the usual way.
+     *
+     * @return array
+     */
+    private function separateLongData()
+    {
+        $streams = $otherData = [];
```

Comment:
Would it make sense to call `$otherData` `$values` same as in the calling method?